src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -106,6 +121,18 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
                 }
             }
 
+            // Check for and attempt to resolve an unknown type before proceeding
+            if (type == null)
+            {
+                // Check to see if the missing type can be resolved (AlsoKnownAs)
+                var unresolvedName = obj["$type"].Value<string>().Split(',').FirstOrDefault();
+                Type newType;
+                nodeFactory.ResolveType(unresolvedName, out newType);
+
+                if (newType != null) // If resolved update the type
+                { type = newType; }
```

Comment:
Here it is preferred to have a single curly brace on a line (_don't know what happened to coding conventions once followed by the team_):
```
// If resolved update the type
if (newType != null) 
{ 
type = newType; 
}
```

Reply:
https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards

Reply:
oh wow, if we would like to get this topic up, please fix all the comments in this PR so they are in the same format 😉 
//[space][Comment]

Reply:
Sorry comments without a space were copied from the constructor I made obsolete, I would never commit such a crime 😂